source/mobile/mobile-faq.rst

```diff
@@ -74,38 +74,38 @@ The following post metadata is sent in all push notifications:
 
 Additional metadata may be sent depending on the System Console setting for `Push Notification Contents <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__:
 
-- **Generic description with sender and channel names**: ``Channel name`` metadata will be included
-- **Full message content sent in the notification payload**: ``Post content`` and ``Channel name`` metadata will be included
+- **Generic description with sender and channel names**: ``Channel name`` metadata will be included.
+- **Full message content sent in the notification payload**: ``Post content`` and ``Channel name`` metadata will be included.
 - **Full message content fetched from the server on receipt** (*available in Enterprise Edition E20*): ``Post content`` and ``Channel name`` are not included in the notification payload, instead the ``Post ID`` is used to fetch ``Post content`` and ``Channel name`` from the server after the push notification is received on the device.
 
 What are my options for securing the mobile apps?
 -------------------------------------------------
 
-The following options for security mobile application deployments are available:
+The following options for secure mobile app deployments are available:
 
 1. Securing network connection to mobile apps
   - Use HTTPS and WSS network connections to encrypt transport.
   - Use of a mobile VPN client on mobile devices to establish secure connection to Mattermost server within private network.
```

Comment:
```suggestion
  - Use of a mobile VPN client on mobile devices to establish secure connection to Mattermost server within a private network.
```

Reply:
@esethna / @DSchalla: do we recommend any specific mobile VPN clients?

Reply:
Never recommend a third party option unless there is some sort of partnership (maybe) which we don't have for VPN clients.